catalog/app/containers/Bucket/Search.js

```diff
@@ -290,15 +312,19 @@ function Search({ bucket, query, page, mode, retry }) {
         >
           <QueryInput query={query || ''} bucket={bucket} onChange={handleQueryChange} />
         </M.Box>
-        <M.Box flexGrow={1} display={{ xs: 'none', sm: 'block' }} />
+        <M.Box
+          flexGrow={1}
+          display={{ xs: 'none', sm: 'block' }}
```

Comment:
i think it doesnt make much sense to keep half of the styles inline and move the other half to the class, just use a div and put all its styles into the class